app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] It will be good for a user to confirm his email before log in, your can achieve this by adding confirmable here

Reply:
Alright, I'll apply that